### ISSUE TITLE
Teleporter exit direction issue

### ISSUE THREAD

**Issue by Krutontar** [mekanism/Mekanism#7085]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*

#### Issue description:
When moving between linked teleporters facing north-south, everything behaves fine but if they face east-west I come out of the destination facing, north-south and run into the frame.

#### Steps to reproduce:
1.  Build teleporters facing E-W direction and observe direction facing on exit.  It's the same coming and going.
2.  If one is facing N-S and the other is facing E-W you will come out the the N-S facing portal correctly but not the other.
3.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 36.1.2
**Mekanism:** 10.0.21.448
**Other relevant version:** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

[[gist](https://gist.github.com/) / [pastebin](https://pastebin.com/) / etc link here. Please make sure that it isn't set to expire.]

**Comment by pupnewfster** [comment]
Build a wall behind the teleporter where you don't want to be coming out.

**Comment by Krutontar** [comment]
There is some strange behavior happening here.  All of the original portals I made before posting this issue were against walls.  I tested a couple east-west facing that were not against walls and they behaved ok.  I added a wall behind 1 and when I enter the other I get turned 90degrees like I described and run into the frame.  Breaking the wall doesn't stop this effect,  I have to break down the portal and rebuild.
